### PR TITLE
Fix some dead links

### PR DESCRIPTION
**Pull request by fgaz**
###### Motivation for this change

As reported by repology: https://repology.org/maintainer/francygazz%40gmail.com/problems

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/gnujump/default.nix [hunk 1]
```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   ];
 
   meta = with stdenv.lib; {
-    homepage = "https://jump.gnu.sinusoid.es/";
```
Comment (veprbl) [inline pkgs/games/gnujump/default.nix]: `"https://jump.gnu.sinusoid.es/"` works perfectly well

Comment (fgaz) [reply pkgs/games/gnujump/default.nix]: It redirects though

Comment (fgaz) [reply pkgs/games/gnujump/default.nix]: (I do not have a strong opinion about this, it's just an url. just tell me if I have to remove that commit)

Comment (Mic92) [reply pkgs/games/gnujump/default.nix]: Should be fine.

**Pull request merged**